tensorflow/python/keras/distribute/multi_worker_callback_tf2_test.py

```diff
@@ -263,7 +293,8 @@ def proc_tensorboard_saves_on_chief_but_not_otherwise(test_obj):
           x=train_ds,
           epochs=num_epoch,
           steps_per_epoch=steps,
-          callbacks=[callbacks.TensorBoard(log_dir=saving_filepath)])
+          # disabling profiler by setting profile_batch to zero
+          callbacks=[callbacks.TensorBoard(log_dir=saving_filepath, profile_batch=0)])
```

Comment:
Can you explain why the need for this change of this line? Thanks!

Reply:
@rchao since profiling result is preserved for both chief and non-chief. This is to disable profiler, otherwise the test will fail.

Reply:
Thanks!